.github/workflows/build-snapshot.yml

```diff
@@ -1,18 +1,25 @@
-name: Dev build and publication
+name: Build and publish jar to sonatype snapshot repository
 
 on:
   push:
-    branches:
-      - '*'
+    branches-ignore:
+      - master
+      - version-*
+    paths-ignore:
+      - README.md
 
 jobs:
-  build:
+  build-job:
     uses: th2-net/.github/.github/workflows/compound-java-dev.yml@main
     with:
       build-target: 'Sonatype'
-      gradleVersion: "8"
+      integration-test-enabled: true
```

Comment:
issue: there are no integration tests to run

Reply:
I have added @Tag annotation into Th2IntegrationTest and configuration for integrationTest task

Reply:
Why would you do that? We should not force any annotations to the users. It is not up to this library to decide what tag should be added on a test

Reply:
For common library, you are right.